yt/frontends/ramses/field_handlers.py

```diff
@@ -448,7 +448,12 @@ def read_rhs(cast):
             # Read nRTvar, nions, ngroups, iions
             for i in range(4):
                 read_rhs(int)
-            f.readline()
+            # Try to read rtprecision
+            try:
+		read_rhs(int)
+		f.readline()
+            except Exception:
+                pass    
```

Comment:
This should fix the tests (replacing tabs with spaces)

```suggestion
            # Try to read rtprecision
            try:
                read_rhs(int)
                f.readline()
            except Exception:
                pass
```